src/libsyntax/ext/tt/transcribe.rs

```diff
@@ -181,6 +187,7 @@ pub fn tt_next_token(r: &mut TtReader) -> TokenAndSpan {
         tok: r.cur_tok.clone(),
         sp: r.cur_span.clone(),
     };
+    //println!("RET {:?} {:?}", r.cur_tok, r.cur_span);
```

Comment:
please remove these commented out `println's`... (or if you think they have long term value, then turn them into uncommented `debug!` invocations.)